bitcoin_executor/sources/interpreter.move

```diff
@@ -671,24 +742,24 @@ fun op_push_small_int_helper(
 ) {
     ip.op_push_small_int(opcode);
     assert_eq!(ip.stack.size(), expected_size);
-    assert_eq!(ip.stack.top(), top_val);
+    assert_eq!(ip.stack.top().destroy_some(), top_val);
     assert_eq!(ip.isSuccess(), true);
 }
 
 #[test]
 fun test_op_equal() {
     let mut ip = new_test_ip(vector[vector[10], vector[10]]);
     ip.op_equal();
-    assert_eq!(ip.stack.top(), vector[1]);
+    assert_eq!(ip.stack.top(), option::some(vector[1]));
 
     ip.stack.push(vector[1]);
     ip.op_equal();
-    assert_eq!(ip.stack.top(), vector[1]);
+    assert_eq!(ip.stack.top(), option::some(vector[1]));
```

Comment:
we can have a helper method `unwrap_top` to make the code easier. It will throw exception if the stack is empty

Reply:
I use destroy_or! macro